python/pyspark/sql/tests/streaming/test_streaming.py

```diff
@@ -26,7 +26,39 @@
 from pyspark.testing.sqlutils import ReusedSQLTestCase
 
 
-class StreamingTests(ReusedSQLTestCase):
+class StreamingTestsMixin:
+    def test_streaming_query_functions_sanity(self):
```

Comment:
Is this a new test?
Could you add brief description? `test... sanity` does not sound good. May be `test_streaming_query_functions_basic()` ?